Assets/Prefabs/Aesthetic Prefabs/OpenDoorwayPrefab.prefab

```diff
@@ -0,0 +1,277 @@
+%YAML 1.1
```

Comment:
Is this just the doorway without the door? or is it something new

Reply:
Yep exactly that. Same model, no door, and added lights on either side